docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.9.1
+    environment:
+      CALYPSO_ENV: 'development'
+    ports:
+      - '3000:3000'
+    volumes:
+      - ./:/calypso
+      - node_modules:/calypso/node_modules
```

Comment:
One of the more interesting things about named volumes, is that if we had a bonafide wp-calypso container, and the container already had built the `node_modules` directory ... it would just copy them from the image to the volume, allowing the user to skip having to `npm install`, unless of course package.json changed ...